db/migrate/20210510092738_add_telegram_onboarding_token.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTelegramOnboardingToken < ActiveRecord::Migration[6.1]
+  def change
+    add_column :contributors, :telegram_onboarding_token, :string
```

Comment:
This should probably have a unique constraint for the unlikely case that the random token we generate already exists, right?